src/Stepper/Stepper.tsx

```diff
@@ -1,7 +1,9 @@
 import * as React from "react"
-
-import { expandColor, OperationalStyleConstants } from "../utils/constants"
+import { OperationalStyleConstants } from "../utils/constants"
+import { useHotkey } from "../useHotkey"
 import styled from "../utils/styled"
+import { YesIcon } from "../Icon/Icon"
+import * as mixins from "../utils/mixins"
```

Comment:
Let's maybe input the only mixin we need: `inputFocus` from here.

```suggestion
import { inputFocus } from "../utils/mixins"
```